integration/ct_integration_local.cfg

```diff
@@ -0,0 +1,26 @@
+[
```

Comment:
Done, but I don't understand the reason you mention (the expanded `[]ct.LogConfig` slice will also contain a bunch of path names, relative to the test).